<?xml version="1.0" encoding="UTF-8"?><ui version="4.0"><class>DiseaseHotspotAnalystDockWidgetBase</class><widget class="QDockWidget" name="DiseaseHotspotAnalystDockWidgetBase"><property name="geometry"><rect><x>0</x><y>0</y><width>320</width><height>400</height></rect></property><property name="windowTitle"><string>Disease Hotspot Analyst</string></property><widget class="QWidget" name="dockWidgetContents"><layout class="QVBoxLayout" name="verticalLayout"><item><widget class="QLabel" name="label"><property name="font"><font><pointsize>10</pointsize><weight>75</weight><bold>true</bold></font></property><property name="text"><string>Filter Outbreaks</string></property></widget></item><item><widget class="QLabel" name="label_2"><property name="text"><string>Disease:</string></property></widget></item><item><widget class="QComboBox" name="diseaseComboBox"/></item><item><widget class="QLabel" name="label_3"><property name="text"><string>Start Date:</string></property></widget></item><item><widget class="QDateEdit" name="startDateEdit"><property name="calendarPopup"><bool>true</bool></property></widget></item><item><widget class="QLabel" name="label_4"><property name="text"><string>End Date:</string></property></widget></item><item><widget class="QDateEdit" name="endDateEdit"><property name="calendarPopup"><bool>true</bool></property></widget></item><item><spacer name="verticalSpacer"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeHint" stdset="0"><size><width>20</width><height>40</height></size></property></spacer></item><item><widget class="QPushButton" name="runAnalysisButton"><property name="styleSheet"><string notr="true">background-color: #27ae60; color: white; padding: 5px; border-radius: 3px;</string></property><property name="text"><string>Run Hotspot Analysis</string></property></widget></item><item><widget class="QLabel" name="statusLabel"><property name="styleSheet"><string notr="true">color: grey;</string></property><property name="text"><string>Status: Ready</string></property></widget></item></layout></widget></widget><resources/><connections/></ui>
